src/app/lib/analyticsUtils/index.test.js

```diff
@@ -1,6 +1,7 @@
 import Cookie from 'js-cookie';
 import { setWindowValue, resetWindowValue } from '@bbc/psammead-test-helpers';
 import onClient from '../utilities/onClient';
+import { getAmpDestination } from './index';
```

Comment:
Imo, it's not good practice to use part of the implementation in the test, better to hardcode the values you expect in the tests and if a regression is introduced to this component, the test will catch it.

Reply:
Yeah I was torn between hardcoding and importing the method, went down the import route for convenience, but if it's better practice I'll hardcode 👍